### ISSUE TITLE
#atldwdg-trigger tag with href="#" reload page

### ISSUE THREAD

**Issue by Korsarro69** [CatalystOne/ngx-jira-issue-collector#1]
**Describe the bug**
When I click on the "a" link tag, the page reload because of the href="#". How do I avoid it ? (It works properly when I remove it on the element editor).

I work with Angular 9.

**Comment by kbjerke** [comment]
@Korsarro69 sorry, unfortunately I did most my testing with a custom trigger. And I see now that the default trigger does not work as intended. For now you can configure the issue collector to use a custom trigger and use:
```html
<div>
  <button (click)="collector.showDialog()">Click me!</button>
  <ngx-jira-issue-collector [configuration]="config" #collector></ngx-jira-issue-collector>
</div>
```

I don't have time to fix this right now.

**Comment by kbjerke** [comment]
@Korsarro69 because of another bug, I prioritized this and released 9.0.2. Hope this fixes your observation.

**Comment by Korsarro69** [comment]
@kbjerke I fixed this issue by removing the attribute href in javascript
